pkg/core/state/validator.go

```diff
@@ -18,6 +18,11 @@ func (vs *Validator) RegisteredAndHasVotes() bool {
 	return vs.Registered && vs.Votes > util.Fixed8(0)
 }
 
+// UnregisteredAndHasNoVotes returns true when Validator is not registered and has no votes.
+func (vs *Validator) UnregisteredAndHasNoVotes() bool {
+	return !vs.Registered && vs.Votes == util.Fixed8(0)
```

Comment:
You can omit `util.Fixed8` cast as int constants are type-less.

Reply:
ACK